examples/bert/tests/smoketest_config_main.yaml

```diff
@@ -63,11 +61,13 @@ optimizer:
 # Training duration and evaluation frequency
 max_duration: 8ba
 eval_interval: 8ba
-eval_subset_num_batches: 8 # For code testing, evaluate on a subset of 8 batches
+# eval_subset_num_batches: 8 # For code testing, evaluate on a subset of 8 batches
```

Comment:
Is this no longer needed? The `eval_interval` above will only run `eval` once at the end of training, but it may still take a long time if the `eval_loader` is big.

Reply:
The tests use a synthetic dataset with 16 examples, so it should be good. Runs fast locally at least.